db/springbootapp_db_script.sql

```diff
@@ -0,0 +1,9 @@
+create table sb_employee1
+(
+  employeeId int auto_increment
+    primary key,
+  firstName  varchar(250) charset utf8 not null,
+  lastName   varchar(250) charset utf8 not null,
+  addedDate  datetime                  not null,
+  status     int                       not null
+);
```

Comment:
Standard practice is to name the columns like so `first_name` instead of `firstName` (unless the client has some really picky preferences)

`employeeId` you can add modifier type unsigned, it is auto-incrementing anyways so the value will never be negative.

`addedDate` even though first thought is to use the DB type datetime, don't. Use `varchar` and insert inside it ISO8601 date as a String.

`status` this has no meaning, make it more meaningful and use an appropriate data type that makes sense. Int can be any number known to man kind.

It is a good idea to have comments in the SQL like  `-- status of 0 means employed and status of 1 means resigned`
Don't use that, but take it as an example

Also instead of defining the charset at the column level just do it at the table level so it applies to everything.

```sql
create table bla_bla_bla (
...
) DEFAULT CHARSET=utf8
```